src/Momento.Sdk.Incubating/ISimpleCacheClient.cs

```diff
@@ -257,6 +257,34 @@ public interface ISimpleCacheClient : Momento.Sdk.ISimpleCacheClient
     /// <returns>Task representing with the status of the fetch operation and the associated set.</returns>
     public Task<CacheSetFetchResponse> SetFetchAsync(string cacheName, string setName);
 
+    /// <summary>
+    /// Push multiple values to the beginning of a list.
+    /// </summary>
+    /// <param name="cacheName">Name of the cache to store the list in.</param>
+    /// <param name="listName">The list to push the value on.</param>
```

Comment:
Can you update these comments for the list concat operation?

Reply:
Fixed in https://github.com/momentohq/client-sdk-dotnet-incubating/pull/78/commits/2892f7416f2e956a386d4df0fa654aecbbf35f89, good catch!